nevergrad/functions/multiobjective/test_core.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import nevergrad as ng
+from nevergrad.functions import MultiobjectiveFunction
+from . import core
+
+
+def test_multiobjective_function() -> None:
+    mfunc = core.MultiobjectiveFunction(lambda x: x, (100, 100))  # type: ignore
+    tuples = [(110, 110), (110, 90), (80, 80), (50, 50), (50, 50), (80, 80), (30, 60), (60, 30)]
+    values = []
+    for tup in tuples:
+        values.append(mfunc(tup))
+    expected = [0, float('inf'), -400, -2500.0, -2500.0, -2500.0, -3300.0, -4100.0]
+    # TODO (80, 80) should yield -2470, and values above the upper bounds could yield positive numbers
```

Comment:
Shoulnt the return value for (80,80) be smaller than 2500?

Reply:
yes as mentioned in the comment it should be -2470; i've fixed this and others.
an enormous thanks, this test is perfect...